experimental/dds/tree2/src/feature-libraries/editable-tree-2/lazyField.ts

```diff
@@ -398,11 +398,15 @@ export class LazyValueField<TTypes extends AllowedTypes>
 		return this.at(0);
 	}
 
+	public set content(newContent: FlexibleNodeContent<TTypes>) {
+		this.setContent(newContent);
+	}
+
 	public get boxedContent(): TypedNodeUnion<TTypes> {
 		return this.boxedAt(0);
 	}
 
-	public setContent(newContent: FlexibleNodeContent<TTypes>): void {
+	private setContent(newContent: FlexibleNodeContent<TTypes>): void {
```

Comment:
I assume if we decide to remove the explicit setter we will merge these methods?

Reply:
Yeah